sagemaker-python-sdk/pytorch_horovod_mpi/pytorch_horovod_mpi.ipynb

```diff
@@ -0,0 +1,791 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training PyTorch Models using Horovod with Open MPI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "Horovod is a distributed deep learning training framework that is used to speed up deep learning workloads. This notebook will show you how to train your PyTorch models using Horovod with Open MPI. Using the Sagemaker Python SDK, we will:\n",
+    "\n",
+    "* Upload a data set to S3\n",
+    "* Train a simple neural network with Horovod\n",
+    "* Deploy a model to a SageMaker endpoint\n",
+    "\n",
+    "You can learn more about Horovod and MPI at the following links:\n",
+    "\n",
+    "* https://horovod.readthedocs.io/en/stable/pytorch.html\n",
+    "* https://horovod.readthedocs.io/en/stable/mpirun.html \n"
+   ]
```

Comment:
I'd restructure this into:
1. overview of Horovod + link to their general docs: https://horovod.readthedocs.io/en/stable/summary_include.html
1. overview of SageMaker. here's an example blurb you can copy/paste: Amazon SageMaker is a fully-managed service that provides developers and data scientists with the ability to build, train, and deploy machine learning (ML) models quickly. Amazon SageMaker removes the heavy lifting from each step of the machine learning process to make it easier to develop high-quality models. The SageMaker Python SDK makes it easy to train and deploy models in Amazon SageMaker with several different machine learning and deep learning frameworks, including PyTorch.
1. "In this notebook, we..."